internal/router/util.go

```diff
@@ -82,7 +82,9 @@ func errorToResp1(err error) (int, interface{}) {
 		status = http.StatusNotFound
 	case params.ErrBadRequest, params.ErrInvalidEntity:
 		status = http.StatusBadRequest
-	case params.ErrForbidden, params.ErrEntityIdNotAllowed:
+	case params.ErrForbidden,
+		params.ErrEntityIdNotAllowed,
+		params.ErrReadOnly:
 		status = http.StatusForbidden
```

Comment:
I feel this is more of a 503 Service Unavailable type situation. It should almost certainly be a 5XX error. There's nothing about the request that means it's forbidden.

Reply:
You're forbidden from writing to the store. It seems that StatusForbidden might be appropriate. 503 doesn't seem quite right to me, as much of the service is actually available.